IonDotnet/IonType.cs

```diff
@@ -47,7 +61,7 @@ public static class IonTypeExtensions
         /// </summary>
         /// <param name="t">IonType enum</param>
         /// <returns>true when the this is a scalar type</returns>
-        public static bool IsScalar(this IonType t) => t > IonType.None && t < IonType.Clob;
+        public static bool IsScalar(this IonType t) => t.Id > IonType.Null.Id && t.Id < IonType.Blob.Id;
```

Comment:
It should be greater or equal to the Null.Id, and it should be less than the Clob, not Blob, Id.

Reply:
Done